### PR TITLE
update tranche charting

### PR DESCRIPTION
**Pull request by Siddharth2207**
## Motivation
Incorrect stack index value for charting variable

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [x] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

File: src/tranche/tranche-space.rain [hunk 1]
```diff
@@ -317,11 +317,11 @@ charts:
         - label: Starting tranche
           value: 0.2.0
           description: 'This strategy starts executing sells at the initial tranche (stack item 0.2.0)'          
-        - label: Initial sell price
-          value: 0.5.3
+        - label: Initial effective sell price
```
Comment (thedavidmeister) [inline src/tranche/tranche-space.rain:320]: this is the final literal io ratio, it's not a derived price

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

File: src/tranche/tranche-space.rain (hunk 1 above)
Comment (Siddharth2207) [reply src/tranche/tranche-space.rain:320]: yes correct, for sell order final io ratio is the effective sell price. Do you mean to say I remove the word 'effective' ?

Comment (Siddharth2207) [reply src/tranche/tranche-space.rain]: resolved.

**Comment by Siddharth2207** [comment]
image
image
image
image
image
image

**Review by thedavidmeister — APPROVED** [review]

**Pull request merged**